app/models/legacy_appeal.rb

```diff
@@ -966,7 +963,7 @@ def ready_for_bva_dispatch?
 
   # uses the paper_trail version on LegacyAppeal
   def latest_appeal_event
-    TaskEvent.new(version: versions.last)
+    TaskEvent.new(version: versions.last) if versions.present?
```

Comment:
```suggestion
    TaskEvent.new(version: versions.last) if versions.any?
```
`present?` works, but I would prefer `any?` since `versions` returns a list.